src/components/currentData.js

```diff
@@ -0,0 +1,77 @@
+import React, { PureComponent, PropTypes } from 'react'
+import { connect } from 'react-redux'
+import * as actions from '../actions/actionIndex'
+import reactCSS from 'reactcss'
+
+const styles = reactCSS({
```

Comment:
See my previous comments from `app.js`. 

1. unnecessary single quotes around `default
1. When specifying pixel value, pass an integer. unnecessary to add `px`. React will add this when you pass an object to the `style` prop.